packages/graphback-core/src/db/dataMapper.ts

```diff
@@ -29,10 +29,19 @@ function getTableId(idField: string, data: any): TableID {
 export const getDatabaseArguments = (modelMap: ModelTableMap, data?: any, fieldMap?: any): TableDataMap => {
   const idField = modelMap.idField;
 
-  //TODO: Map fields to custom db names
-
   return {
     idField: getTableId(idField, data),
-    data
+    // eslint-disable-next-line @typescript-eslint/tslint/config
+    data: Object.keys(data).reduce((dataObj, key) => {
```

Comment:
We have to check for if data is defined here. Otherwise it will crash. 

Since from https://github.com/aerogear/graphback/pull/1574/files#diff-5a1ded10109278ede19b3429e917d253R29 data can be undefined.

Reply:
I'll push a commit to only transform data if defined.

Reply:
This was the case for the MongoDB provider https://github.com/aerogear/graphback/blob/master/packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts#L130 causing the Mongo integration tests to fail.